lib/puppet/provider/service/init.rb

```diff
@@ -72,6 +72,7 @@ def self.excludes
     excludes += %w{cryptdisks-udev}
     excludes += %w{statd-mounting}
     excludes += %w{gssd-mounting}
+    excludes
```

Comment:
This looks like it snuck in to 30cc5672af41adbbe2cdc12b916256be2f081a1e which only talks about ignoring unused variables?

Reply:
That should have been added to https://github.com/puppetlabs/puppet/pull/6446/commits/bf2e95afd2db4a1cf6ca39a5e25463513194ae46. I'll move it.